requirements-test.txt

```diff
@@ -5,3 +5,4 @@ channels_redis~=4.1.0
 pytest-asyncio~=0.21.0
 pytest-django~=4.5.0
 freezegun~=1.2.2
+selenium~=4.18.1
```

Comment:
We already depend on selenium in openwisp-utils[selenium], we may want to look if we can avoid redefining this here: https://github.com/openwisp/openwisp-utils/blob/master/setup.py#L67